src/components/SEO.js

```diff
@@ -7,25 +7,39 @@ import config from '../../config';
 const SEO = props => {
   const { postNode, postPath, article, buildTime } = props;
 
-  let title;
-  let description;
-  let keywords;
-
   const realPrefix = config.pathPrefix === '/' ? '' : config.pathPrefix;
   const homeURL = `${config.siteUrl}${realPrefix}`;
   const URL = `${homeURL}${postPath || ''}`;
-  const image = `${homeURL}${config.siteBanner}`;
+
+  //Set defaults
+  //Will attempt to chnage based on postNode.frontMatter
+  let title = config.siteTitleAlt;
+  let description = config.siteDescription;
+  let keywords =
+    'Donate medical masks, covid19, homemade masks, homemade surgical mask, surgical mask, reuseable masks';
+  let image = `${homeURL}${config.siteBanner}`;
 
   if (article) {
     const postMeta = postNode.frontmatter;
     title = `${postMeta.title} | ${config.siteTitle}`;
-    description = postNode.excerpt;
-    keywords = postNode.frontmatter.keywords.join();
-  } else {
-    title = config.siteTitleAlt;
-    description = config.siteDescription;
-    keywords =
-      'Donate medical masks, covid19, homemade masks, homemade surgical mask, surgical mask, reuseable masks';
+    //Description may be set, else, use excerpt
+    if (postMeta.hasOwnProperty('description')) {
+      description = postMeta.description;
+    } else if (postNode.hasOwnProperty('excerpt')) {
+      description = postNode.excerpt;
```

Comment:
I think one of these isn't working quite right. If you look at the twitter card validator for the preview site - the only card with a description now is the `patterns` page. So I think one of those two checks aren't actually working as they are _always_ overwriting the siteDescription.

Reply:
>  I think one of those two checks aren't actually working as they are always overwriting the siteDescription.

If the post has frontmatter.description, _it is_ overwriting  `siteDescription`